lidarr/command.go

```diff
@@ -82,3 +85,24 @@ func (l *Lidarr) SendCommandContext(ctx context.Context, cmd *CommandRequest) (*
 
 	return &output, nil
 }
+
+// GetCommandStatus returns the status of an already started command.
+func (s *Lidarr) GetCommandStatus(commandID int64) (*CommandResponse, error) {
```

Comment:
the `s` needs to be `l`.

Reply:
Yeah, I pulled it down and ran make test ;)